quinn-proto/src/connection/mtud.rs

```diff
@@ -409,27 +575,42 @@ mod tests {
     }
 
     #[test]
-    fn mtu_discovery_disabled_lost_four_packets_triggers_black_hole_detection() {
+    fn mtu_discovery_disabled_lost_four_packet_bursts_triggers_black_hole_detection() {
         let mut mtud = MtuDiscovery::disabled(1_400);
         let now = Instant::now();
 
-        for _ in 0..4 {
-            mtud.on_non_probe_lost(now, 0, 1300);
+        for i in 0..4 {
```

Comment:
Not a new issue, but should this for loop and similar below be defined in terms of `BLACK_HOLE_THRESHOLD` rather than hardcoded?

Reply:
I actually like the fact that the testing code is not tied to the constant. That way tests will fail if someone were to accidentally increase the threshold. Also, there is a chance we will make the threshold configurable in the future (if it turns out to be necessary to tweak), in which case there will be no constant to use.